ovn-fake-multinode-utils/playbooks/deploy-minimal.yml

```diff
@@ -42,3 +42,9 @@
       src: "{{ rundir }}/log-collector.sh"
       dest: /tmp/log-collector.sh
       mode: '0744'
+
+  - name: Deploy perf script
+    copy:
+      src: "{{ rundir }}/perf.sh"
```

Comment:
This fails because `perf.sh` is not in `{{ rundir }}`. We need to also change the `generate()` function in `do.sh` to copy the script to the runtime dir. Similar as for log-collector:
```
cp ${ovn_fmn_utils}/scripts/log-collector.sh ${log_collector_file}
```

Reply:
@dceara ack, thx. Fixes now